### PR TITLE
Added VMsStatusReport file

### PR DESCRIPTION
**Pull request by jimitkg**
Added a new file that creates a color coded, css styled VM status report utilizing enhanced-html2 module. This script is tested against vmware 6.7.  Here is a snapshot of example output.  https://github.com/jimitkg/vmreports/blob/master/VMStatusReport-Example.png

### CODE REVIEW

**Review by dmilov — APPROVED** [review]
Looks good, some small code style issues has to be addressed before merge

File: Scripts/VMsStatusReport.ps1 [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<#
+Script name: VMsStatusReport.ps1
+Created on: 20/06/2021
+Author: Jimit Gohel, @PsJimKG
+Description: The purpose of the script is to get a list of all VM Snaphots, Powered On VMs and Powered Off VMs. Utlises css and converts to a good looking html report. HTML format highlights large/old snapshots. This can be scheduled via taks scheduler and emailed to administrators or uplaoded to an IIS web location.
```
[the remark this answers is not in the document]
Comment (jimitkg) [reply Scripts/VMsStatusReport.ps1:5]: Thank you

File: Scripts/VMsStatusReport.ps1 [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Dependencies: Along with PowerCli, this script also requires Don Jones EnhancedHTML2 module. https://www.powershellgallery.com/packages/EnhancedHTML2/2.1.0.1
+
+===Tested Against Environment====
+vSphere Version: 6.7
+PowerCLI Version: PowerCLI 12.0
+PowerShell Version: 5.1
+OS Version: Windows 10
+
+#>
+$viservers = "vcenter1", "vcenter2"
```
[the remark this answers is not in the document]
Comment (jimitkg) [reply Scripts/VMsStatusReport.ps1]: Great suggestion. I will convert this script to a function with VIServersFilePath and OutPath parameter and create a new pull request.

File: Scripts/VMsStatusReport.ps1 [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+#Capturing data for snapshots
+
+    $paramsSnapshotFragment = @{'As'='Table';
+            'PreContent'='<h2>VM Snapshots</h2>';
+            'MakeTableDynamic'=$true;
+            'TableCssClass'='grid';
+            'Properties' = 'VM',@{Name = 'SnapshotName';Expression = {$_.Name}},'Created', @{Name = 'AgeDays';Expression = {((New-TimeSpan -Start $_.Created -End (get-date)).Days)};css={if (((New-TimeSpan -Start $_.Created -End (get-date)).Days) -gt 120) {'red'}}},'Description', @{name = 'SizeGB';expression = {[math]::Round($_.sizegb,2)};css={if ($_.sizegb -gt 100) {'red'}}}, 'PowerState','IsCurrent','ParentSnapshot','Children'
+    }
+    $snapshots = $vms |Get-Snapshot |ConvertTo-EnhancedHTMLFragment @paramsSnapshotFragment
```
[the remark this answers is not in the document]
Comment (jimitkg) [reply Scripts/VMsStatusReport.ps1]: accepted. Thank you

File: Scripts/VMsStatusReport.ps1 [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+#PoweredOn
+
+$paramsPoweredon = @{'As'='Table';
+'PreContent'='<h2>Powered On VMs</h2>';
+'MakeTableDynamic'=$true;
+'TableCssClass'='grid';
+'Properties' = 'Name','PowerState', 'Notes' , 'Folder'
+}
+$poweredonvms = $vms |Where-Object {$_.PowerState -eq 'PoweredOn'}|ConvertTo-EnhancedHTMLFragment @paramsPoweredon
```
[the remark this answers is not in the document]
Comment (jimitkg) [reply Scripts/VMsStatusReport.ps1]: thank you

**Review by jimitkg — COMMENTED** [review]
Great feedback. New pull request will be created with suggested changes.

**Review by dmilov — APPROVED** [review]

**Pull request merged**